pkg/core/blockchain.go

```diff
@@ -651,12 +651,17 @@ func (bc *Blockchain) storeBlock(block *block.Block) error {
 						continue
 					}
 					op, ok := arr[0].Value().([]byte)
-					if !ok || string(op) != "transfer" {
+					if !ok || (string(op) != "transfer" && string(op) != "Transfer") {
 						continue
 					}
-					from, ok := arr[1].Value().([]byte)
-					if !ok {
-						continue
+					var from []byte
+					fromValue := arr[1].Value()
+					// we don't have `from` in native contracts
+					if fromValue != nil {
+						from, ok = fromValue.([]byte)
+						if !ok {
+							continue
+						}
 					}
 					to, ok := arr[2].Value().([]byte)
```

Comment:
And by the way this could happen here too because we can also burn tokens.

Reply:
Added check for `to`.